com.ibm.streamsx.inet/impl/java/src/com/ibm/streamsx/inet/http/HTTPPostOper.java

```diff
@@ -389,6 +422,23 @@ void sleepABit(double seconds) throws InterruptedException {
 			Thread.sleep(100);
 		}
 	}
+	
+	boolean isAttributeToPost(String attributeName) {
+		
+		if (postAttributes.isEmpty()) {
+			trace.log(TraceLevel.INFO, "postAttriburte to use: " + attributeName);
+			return true;
+		}
+		for (Iterator<String> iterator = postAttributes.iterator(); iterator.hasNext();) {
+			String postAttribute = iterator.next();
+			if (postAttribute.equals(attributeName)) {
+			        trace.log(TraceLevel.INFO, "postAttriburte to use: " + attributeName);
```

Comment:
This tracing (and line 439) is executed on every tuple but is fixed per scheme.
It's far too expensive and worthless to do every tuple. 

An info trace during initialization on the attributes that will be posted is sufficient.